prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1578,6 +1577,28 @@ void rxr_ep_progress(struct util_ep *util_ep)
 	fastlock_release(&ep->util_ep.lock);
 }
 
+static
+bool rxr_ep_use_shm(struct fi_info *info)
+{
+	/* App specified requires supercede environmental variables
```

Comment:
Nit: This sentence seems off. I'm guessing you meant app hints?

Reply:
Yes, I meant app provided hints. fixed in new revision.

Reply:
Discussed with Wei offline. I was mostly thinking about the function signatures of `rxr_pkt_post_ctrl_or_queue()`,  `rxr_pkt_post_ctrl()`, and `rxr_pkt_post_ctrl_once()` and the inject flag on them, but we can rework that outside this PR. Seems like an artifact of merging the shm fallback and EFA rendezvous protocol.